applications/luci-app-mwan3/root/usr/share/luci/menu.d/luci-app-mwan3.json

```diff
@@ -0,0 +1,105 @@
+{
+	"admin/status/mwan3": {
+		"title": "MultiWAN Manager",
+		"order": "600",
+		"action": {
+			"type": "alias",
+			"path": "admin/status/mwan3/overview"
+		},
+		"depends": {
+			"acl": [ "luci-app-mwan3" ]
+		}
+	},
+	"admin/status/mwan3/overview": {
+		"title": "Overview",
+		"order": 10,
+		"action": {
+			"type": "view",
+			"path": "mwan3/status/overview"
+		}
+	},
+	"admin/status/mwan3/detail": {
+		"title": "Status",
+		"order": 20,
+		"action": {
+			"type": "view",
+			"path": "mwan3/status/detail"
+		}
+	},
+	"admin/status/mwan3/diagnostics": {
+		"title": "Diagnostics",
+		"order": 30,
+		"action": {
+			"type": "view",
+			"path": "mwan3/status/diagnostics"
+		}
+	},
+	"admin/status/mwan3/troubleshooting": {
+		"title": "Troubleshooting",
+		"order": 40,
+		"action": {
+			"type": "view",
+			"path": "mwan3/status/troubleshooting"
+		}
+	},
+
+	"admin/network/mwan3": {
+		"title": "MultiWAN Manager",
+		"order": "600",
+		"action": {
```

Comment:
You could use action type "firstchild" here to avoid the need to repeat the `admin/network/mwan3/globals` path.